nasl/nasl_ssh.c

```diff
@@ -222,6 +222,8 @@ extern int lowest_socket;
  *
  * - @a scciphers SSH server-to-client ciphers.
  *
+ * - @a timeout Set a timeout for the connection in seconds.
```

Comment:
```suggestion
 * - @a timeout Set a timeout for the connection in seconds. Defaults to 10 seconds (defined by libssh internally) if not given.
```

Found the following in https://github.com/libssh/libssh-mirror/blob/cd15043656280e92714c139d820ef271d00876a4/src/misc.c#L1372-L1373:
```
		res = 10 * 1000; /* use a reasonable default value in case
				* SSH_OPTIONS_TIMEOUT is not set in options. */
```